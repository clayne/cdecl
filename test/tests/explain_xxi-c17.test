EXPECTED_EXIT=65
cdecl -xc17 <<END
explain constexpr int x
END
