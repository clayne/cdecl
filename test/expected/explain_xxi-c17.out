explain constexpr int x
        ^
9: error: "constexpr": unsupported keyword until C23
